src/Components/Navigation.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FaUser } from 'react-icons/fa';
+
+const Navigation = () => (
+  <>
+    <div className="nav-bar">
+      <div className="logo">
+        <p className="magicians">Bookstore CMS</p>
+      </div>
+      <nav className="nav">
+        <Link className="links" style={{ textDecoration: 'none' }} to="/">Books</Link>
+        <Link className="links" style={{ textDecoration: 'none' }} to="/categories">Categories</Link>
+      </nav>
```

Comment:
- [ **OPTIONAL** ] **RECOMMNDED:** As this was not pointed put by the previous reviewer, kindly make sure you don't use inline styling. But if you want to use React inline styling you can create a variable should be created and passed to React style.
```JavaScript
const myComponentStyle = {
  color: 'blue',
 ...
}

function MyComponent(){

    return <div style={myComponentStyle}> Inline Styled Component</div>

}
```

**CONSISTENCY:** Kindly make sure to use the same pull request for submitting your project till it is approved.